snuba/cli/devserver.py

```diff
@@ -153,6 +153,7 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                     "--auto-offset-reset=latest",
                     "--log-level=debug",
                     "--consumer-group=metrics_group",
+                    "--dead-letter-topic=snuba-dead-letter-inserts",
```

Comment:
Why this ? we are failing the entire batch if there is a failure on the metrics consumer. We should not need a dead letter queue (at least not this dead letter queue) there .